src/subdomains/supporting/support-issue/entities/support-issue.entity.ts

```diff
@@ -28,12 +29,18 @@ export class SupportIssue extends IEntity {
   @Column({ length: 256, nullable: false })
   reason: SupportIssueReason;
 
+  @Column({ length: 256, nullable: true })
+  name: string;
```

Comment:
What is this for?

Reply:
To have a name for the user, like in other support ticket systems

Reply:
Then we should maybe rename it to `user`? Or `userName`? And I didn't find a usage of this field.